.github/workflows/lintbuildandtest.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         run: npm run lint:strict
       - name: Test
         id: test
-        run: firebase emulators:exec --only auth,firestore 'npm --prefix functions run test:ci'
+        run: npm run test:ci
```

Comment:
Wondering if we should re-enable the tests with the emulator now that we have identified the fix?

Reply:
This is running with the emulators - I just moved this part into the outermost package.json as a script